sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py

```diff
@@ -215,13 +215,18 @@ def __init__(self, codes):
                 RegBitField(consts.TX_DISABLE_SUPPORT_FIELD, 1),
                 size=2, format="<H"
             ),
+            NumberRegField(consts.CTRLS_ADVT_FIELD, self.getaddr(0x1, 156),
+                RegBitField(consts.RX_DISABLE_SUPPORT_FIELD, 1),
+                size=2, format="<H"
+            ),
             NumberRegField(consts.TX_FLAGS_ADVT_FIELD, self.getaddr(0x1, 157),
                 RegBitField(consts.TX_FAULT_SUPPORT_FIELD, 0),
                 RegBitField(consts.TX_LOS_SUPPORT_FIELD, 1),
                 RegBitField(consts.TX_CDR_LOL_SUPPORT_FIELD, 2),
                 RegBitField(consts.TX_ADAPTIVE_INPUT_EQ_FAIL_FLAG_SUPPORTED, 3),
             ),
             NumberRegField(consts.RX_FLAGS_ADVT_FIELD, self.getaddr(0x1, 158),
+                RegBitField(consts.RX_FAULT_SUPPORT_FIELD, 0),
```

Comment:
@vdahiya12 I don't see this defined. Can we please remove this?
![image](https://github.com/user-attachments/assets/7d0341e2-1065-4121-9905-5b15dc92a1f7)

Reply:
fixed